ui/app/common/restclient/services/ReportDAO.js

```diff
@@ -44,6 +44,20 @@
         });
       };
 
+      resource.getProductsByGAV = function (groupId, artifactId, version) {
+        return Configuration.then(function (config) {
+          return $http.get(config.dependencyAnalyzerReportsURL + '/v-0.4/listings/whitelist/artifacts/gav', {
```

Comment:
What about keeping all the endpoints in the config.json file, like getWhitelistProductArtifacts ?